Edited installing swift index (#649)
apple/swift-org-website: install/index.md

title: Install Swift
---

Whether you are a beginner or an experienced programmer, learning and using Swift can open up opportunities for you to build innovative applications for Apple devices and applications beyond Apple's ecosystem. Not only can you use Swift on macOS, but using Swift on Linux and Windows makes Swift a valuable language for developers looking to work across different platforms and environments. 

Follow these general steps to start using Swift:

> 1. Choose your development platform below and follow the instructions to install the latest version of Swift. See [supported platform](/platform-support) for the level of help for each platform.

Reviewer: ```suggestion
1. Choose your development platform below and follow the instructions to install the latest version of Swift. See the [supported platforms page](/platform-support) for the platforms that Swift supports.
```

Author: This is a little redundant. Maybe say something as simple as, "See the [supported platforms page](/platform-support) for more information." wdyt?

> 2. After the installation is complete, you can verify the installation by running the swift —version command.

Reviewer: ```suggestion
2. After the installation is complete, you can verify the installation by running `swift —version`.
```

Author: good catch!
